fb_instant_articles.module

```diff
@@ -18,6 +18,16 @@ function fb_instant_articles_menu() {
     'file path' => drupal_get_path('module', 'system'),
   );
 
+  $items['admin/config/services/fb-instant-articles/settings'] = array(
```

Comment:
@scottrigby So sorry. I had cleared my menu cache, but it turns out I had messed up my sandbox and had a previous version of the module code lying around. Stupid.

I actually think the behavior on the Modules page made more sense before (having the Configure link lead to the interstitial `admin/config/services/fb-instant-articles`). Would you be up for reverting that first hunk of ad92018 and keep the second one?